code/modules/reagents/chemistry/reagents/water.dm

```diff
@@ -373,6 +381,7 @@
 	process_flags = ORGANIC | SYNTHETIC		//Admin-bus has no brakes! KILL THEM ALL.
 	metabolization_rate = 1
 	can_synth = 0
+	taste_message = null
```

Comment:
I...dunno! Hellwater cookies are used surprisingly often, but maybe a part of the adventure is not knowing?

I'll set it to "admin abuse" for now. For the keks.